core/src/main/java/org/apache/iceberg/AllManifestsTable.java

```diff
@@ -44,15 +44,16 @@
       Types.NestedField.required(1, "path", Types.StringType.get()),
       Types.NestedField.required(2, "length", Types.LongType.get()),
       Types.NestedField.optional(3, "partition_spec_id", Types.IntegerType.get()),
-      Types.NestedField.optional(4, "added_snapshot_id", Types.LongType.get()),
-      Types.NestedField.optional(5, "added_data_files_count", Types.IntegerType.get()),
-      Types.NestedField.optional(6, "existing_data_files_count", Types.IntegerType.get()),
-      Types.NestedField.optional(7, "deleted_data_files_count", Types.IntegerType.get()),
-      Types.NestedField.optional(8, "partition_summaries", Types.ListType.ofRequired(9, Types.StructType.of(
-          Types.NestedField.required(10, "contains_null", Types.BooleanType.get()),
-          Types.NestedField.required(11, "contains_nan", Types.BooleanType.get()),
-          Types.NestedField.optional(12, "lower_bound", Types.StringType.get()),
-          Types.NestedField.optional(13, "upper_bound", Types.StringType.get())
+      Types.NestedField.optional(4, "content", Types.IntegerType.get()),
```

Comment:
The correct way to add a new field in this manifest table is :  add the `content` field with an auto-increment field id (which should be 14 in this case) at the tail of those nested field list. We cannot change the existing field ids in this schema because them are mapping to the underlying rows in generated avro files.

Reply:
OK, I will fix it. Thanks.

Reply:
Prefer to add it as the last entry so that ID of other fields are not notified for backwards compatibility.